curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/64367cb9facaa32bacfbff2b.md

```diff
@@ -0,0 +1,120 @@
+---
+id: 64367cb9facaa32bacfbff2b
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+# --description--
+
+Before starting the project, you need to set up the HTML document with some basic structure. Imagine the web page as a house, and HTML as the foundation. You'll need a strong foundation, so let's add a `DOCTYPE`, `html`, `head`, and `body` tags. Think of them as the pillars that hold up the roof of the website. 
```

Comment:
Only the first two JavaScript projects should have campers write the HTML, CSS and JavaScript.

All of the other projects should only have campers focus on the JavaScript. The first step should be a JavaScript focused step. 

The reason for this is because campers will have gone through the entire responsive web design course and will have plenty of practice with HTML and CSS. We want to make sure to focus on the JavaScript only.

Reply:
removed HTML CSS steps